ktor-http/ktor-http-cio/common/src/io/ktor/http/cio/internals/Tokenizer.kt

```diff
@@ -19,7 +19,7 @@ internal fun skipSpacesAndHorizontalTabs(
     var index = start
     while (index < end) {
         val ch = text[index]
-        if (ch != ' ' && ch != HTAB) break
+        if (!ch.isWhitespace() && ch != HTAB) break
```

Comment:
Isn't it too slow to use (it is much slower that just cmpeq because of double table lookup and indirection)? Why should we ignore all unicode whitespaces here?